{{ cookiecutter.project_slug }}/Makefile

```diff
@@ -17,12 +19,15 @@ clean: ## delete most git-ignored files
 	find . -name '*~' -exec rm -f {} +
 
 requirements: ## install environment requirements
-	pip install -r requirements.txt
+	pip install -r requirements/base.txt
+
+compile-requirements: ## compile requirements files
```

Comment:
We already have the `upgrade` target

Reply:
This command is to compile the current requirements, not to upgrade dependencies. @morenol

Reply:
Ok, in which cases will we be interested in to compile instead of upgrade the dependencies?

Reply:
When adding new dependencies. @morenol